content/en/infrastructure/containers/configuration.md

```diff
@@ -190,11 +190,61 @@ Set the environment variable on both the Process Agent and Cluster Agent contain
 {{% /tab %}}
 {{< /tabs >}}
 
+### Collect custom resources and CustomResourceDefinitions
+
+The [Orchestrator Explorer][3] collects CustomResourceDefinitions by default. These definitions appear in Datadog without any user configuration required.
+
+To collect custom resources, you need to configure both the Datadog Agent and set up indexing.
+
+1. Configure the Datadog Agent:
+
+   {{< tabs >}}
+   {{% tab "Helm Chart" %}}
+
+   Add the following configuration to `datadog-values.yaml`:
+
+   ```
+   orchestratorExplorer:
+       customResources:
+           - <CUSTOM_RESOURCE_NAME>
+   ```
+
+   Each `<CUSTOM_RESOURCE_NAME>` must use the format `group/version/kind`.
+
+   {{% /tab %}}
+   {{% tab "Datadog Operator" %}}
+
+   The Datadog Operator needs permission to collect custom resources. Install the Operator with an option that grants this permission:
```

Comment:
the operator isn't doing the collecting, it's granting permissions to the (Cluster) Agent to do the collecting

```suggestion
   The Datadog Operator needs permission to allow the Agent to collect custom resources. Install the Operator with an option that grants this permission:
```